### ISSUE TITLE
Integration with Preact currently works, however it throws an error in the console.

### ISSUE THREAD

**Issue by schalkventer** [schalkventer/react-html-connector#1]
Replicated at: https://s.codepen.io/schalkventer/debug/MXZErW

```
Uncaught DOMException: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.
    at h (https://cdnjs.cloudflare.com/ajax/libs/preact/8.2.9/preact.min.js:1:2866)
    at M (https://cdnjs.cloudflare.com/ajax/libs/preact/8.2.9/preact.min.js:1:7615)
    at https://unpkg.com/react-html-connector@0.1.5/dist/index.min.js:1:2428
    at Array.forEach (<anonymous>)
    at t.default (https://unpkg.com/react-html-connector@0.1.5/dist/index.min.js:1:2383)
    at e.value (https://unpkg.com/react-html-connector@0.1.5/dist/index.min.js:1:6015)
    at https://s.codepen.io/schalkventer/debug/MXZErW:156:17
```

**Comment by schalkventer** [comment]
Tried fixing in accordance with https://github.com/developit/preact/issues/24 in commit 0da2b2d60b6a741713e02daa17e449732ece313d.

**Comment by schalkventer** [comment]
Fixed in 88f9c992905a6e61298eef39afece9ea60f8b5ff
